core/server/web/shared/middlewares/cache-clear.js

```diff
@@ -0,0 +1,32 @@
+// # CacheClear Middleware
+// Usage: Automatic
+// After: cacheControl
+// Before: routes
+// App: Admin|Site|API
+//
+// Allows deletion of files stuck in cache
+const fs = require('fs-extra');
+const common = require('../../../lib/common');
+
+const deleteSingleFile = file => fs.unlink(file.path).catch(err => common.logging.error(err));
+
+const cacheClear = (req, res, next) => {
```

Comment:
Not sure about naming here (and module file). `cache` is something that is used repeatedly, and what we are dealing here with is just a temporary file storage. What do you think about `uploadClear` or any other ideas?